modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -24,8 +27,9 @@ rules:
   resources: ["secrets"]
   verbs: ["*"]
 ----
-. Click *Create* to add the cluster role.  
-. Now create the cluster role binding. In the web console, select *User Management* -> *Role Bindings* -> *Create Binding*.
+
+. Click *Create* to add the cluster role.
+. Now create the cluster role binding. In the main menu, again select *User Management* -> *Role Bindings* -> *Create Binding*.
```

Comment:
I don't think it's necessary to call out that menu. Maybe:
```suggestion
. To create the cluster role binding, select *User Management* -> *Role Bindings* -> *Create Binding*.
```

Reply:
ACK.

Addressed